conans/model/conf.py

```diff
@@ -69,6 +69,8 @@
     "tools.deployer:symlinks": "Set to False to disable deployers copying symlinks",
     "tools.files.download:retry": "Number of retries in case of failure when downloading",
     "tools.files.download:retry_wait": "Seconds to wait between download attempts",
+    # TODO. Find better name
+    "tools.graph:skip_binaries": "Lorem Ipsum",
```

Comment:
Yeah, probably we want to negate it and be ``force_install=True``, ``dont_skip=True``?
I don't oppose to that name, it is not that bad ``tools.graph:skip_binaries`` doesn't sound bad.

Reply:
Let's go with the already present one then :)